src/graph_notebook/notebooks/04-Machine-Learning/Neptune-ML-SPARQL/neptune-ml-pretrained-model-config.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Do we need a different file for this or can we leverage the other one?

Does this use different containeres?

Reply:
I'm pretty sure we need to build new...